libkbfs/test_stallers.go

```diff
@@ -157,6 +157,7 @@ func (s *NaïveStaller) StallMDOp(stalledOp StallableMDOp, maxStalls int,
 			},
 			delegate: jServer.delegateMDOps,
 		}
+		s.config.SetMDOps(jServer.mdOps())
```

Comment:
Sorry I forgot to mention this in the PR summary.  `config.MDOps()` is a reference type, not a pointer.  So callers of it (in this case, the FBO background fetcher) still see an instance with the old, non-stallable delegate.